### PR TITLE
Indentation and spacing polishing

### LINKED ISSUE #109: Indentation is not preserved with enumerations
As discovered in ISI analysis.

### PR DESCRIPTION
**Pull request by jvcoutinho**
Fixes #109 and #103.

- Fixes indentation of conflicts generated by S3M (a corner case).
- Adds tests for imports spacing.

### CODE REVIEW

File: src/main/java/br/ufpe/cin/generated/SimplePrintVisitor.java [hunk 1]
```diff
@@ -175,18 +190,6 @@ public boolean visit(FSTNonTerminal nonTerminal) {
 		if (nonTerminal.getType().equals("EnumConstants")) {
 			printFeatures(nonTerminal,true);
 			Iterator<FSTNode> listElements = getChildren(nonTerminal, "EnumConstant").iterator();
-
-			// fix commas
```
Comment (guilhermejccavalcanti) [inline src/main/java/br/ufpe/cin/generated/SimplePrintVisitor.java]: @jvcoutinho esse fix hardcoded não é mais necessário?

File: grammars/java18_merge_fst.gcide [hunk 2]
```diff
@@ -468,18 +468,33 @@ ClassOrInterfaceTypeAnnotation:
...
 @FSTInline 
 EnumDeclaration:  
-        "enum" Id
+        Enum Id
         [ ImplementsList ]
         EnumBody
 ;
 
+@FSTTerminal(merge="LineBased")
+Enum:
```
Comment (guilhermejccavalcanti) [inline grammars/java18_merge_fst.gcide]: @jvcoutinho por que foi necessário criar uma regra só para uma keyword?

Comment (jvcoutinho) [reply grammars/java18_merge_fst.gcide]: Caso contrário, a indentação da keyword `enum` era ignorada.

A keyword `class` se comporta do mesmo jeito.

File: src/main/java/br/ufpe/cin/generated/SimplePrintVisitor.java (hunk 1 above)
Comment (jvcoutinho) [reply src/main/java/br/ufpe/cin/generated/SimplePrintVisitor.java]: Esse hardcoded servia para consertar as vírgulas, certo?

No teste que fiz não vi problema.
